### DISCUSSION TITLE
how to use custom cursors with model-viewer

### DISCUSSION THREAD

**Discussion by orija** [google/model-viewer#2179] [Q&A]
Hello,

Like the title says, I am hoping to use custom image cursors with model viewer. When I change the cursor: css in the modelviewer script it works, but simply changing the css style on my  html page will not. I'm wondering how I could access the element without altering the script.

with javascript?

any help appreciated,
Au-

**Comment by orija** [comment]
and where on earth is the interaction-prompt cursor specified? (so i can change it.)

**Reply by elalish** [reply]
You can provide a custom interaction prompt with a slot: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-slots-interactionPrompt

**Reply by orija** [reply]
Thanks. That works!
